BrowserKit/Sources/ToolbarKit/AddressToolbar/LocationView/LocationTextField.swift

```diff
@@ -50,6 +50,9 @@ class LocationTextField: UITextField, UITextFieldDelegate, ThemeApplicable {
         autocapitalizationType = .none
         returnKeyType = .go
         tintAdjustmentMode = .normal
+
+        // Disables OS to load AutoFill content, since for URL type is not needed.
```

Comment:
What does OS stand for here?

Reply:
Oh, is this about the OS Keyboard? Probably should just add that in here and then it's clearer